code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -133,8 +132,9 @@
 
 /datum/symptom/heal/coma/proc/coma(mob/living/M)
 	if(deathgasp)
-		M.emote("deathgasp")
-	M.fakedeath("regenerative_coma")
+		M.fakedeath("regenerative coma")
+	else
+		M.Unconscious(300)
```

Comment:
Cant you get around this with unconscious preventing reagents / NOSLEEP on the species?

Reply:
Well yes, but Unconcious() has a bypass for that in it's arguments set by default to FALSE. I will just call it with bypass as true.